internal/constants/constants.go

```diff
@@ -71,6 +71,10 @@ const (
 	LogStoreURL                = "https://" + ElasticsearchFQDN + ":" + ElasticsearchPort
 	MasterCASecretName         = "master-certs"
 	CollectorSecretName        = "collector"
+	PodSecurityLabelKey        = "pod-security.kubernetes.io/enforce"
+	PodSecurityLabelKey2       = "pod-security.kubernetes.io/audit"
+	PodSecurityLabelKey3       = "pod-security.kubernetes.io/warn"
```

Comment:
rename variables to reflect label names. `PodSecurityLabelEnforce`, `PodSecurityLabelAudit`, `PodSecurityLabelWarn`